analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PreferGuidEmpty.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+
+class Compliant
+{
+    Guid? NullableField; // Compliant
+    readonly Guid? ReadonlyNullableField; // Compliant
+    Guid Property { get; set; } // Compliant
+
+    void Empty()
+    {
+        var empty = Guid.Empty; // Compliant
+    }
+
+    void NotEmpty()
+    {
+        var rnd = Guid.NewGuid();// Compliant
+        var bytes = new Guid(new byte[0]); // Compliant
+        var str = new Guid("FA97FFE7-532C-4015-8698-49D8CE4126F4"); // Compliant
+    }
+
+    void NullableDefault()
+    {
+        Guid? nullable = default; // Compliant, not equivalent to Guid.Empty.
+        var nullableOfT = default(Guid?); // Compliant
+        var instance = new NullableGuidClass(default); // Compliant
+        instance.Method(default); // Compliant
```

Comment:
In general, it makes the comments easier to read when they are (partially) aligned with surrounding rows.
```suggestion
        Guid? nullable = default;                                    // Compliant, not equivalent to Guid.Empty.
        var nullableOfT = default(Guid?);                       // Compliant
        var instance = new NullableGuidClass(default); // Compliant
        instance.Method(default);                                   // Compliant
```